sdkutil/event.go

```diff
@@ -16,6 +18,9 @@ var (
 	ErrUnknownModule = errors.New("Unknown module")
 	// ErrUnknownAction is the error with message "Unknown action"
 	ErrUnknownAction = errors.New("Unknown action")
+
+	// EventTypeMessage defines the Akash message string
+	EventTypeMessage = akashEventMessageV1
```

Comment:
any reason why this isn't a constant?

Reply:
It references the set constant. 

Reasoning: In the potential case we need to switch to `akash.v2` the `const` values don't need to change, we just change what `EventTypeMessage` references, and we can still reference current and prior const values within the package.

Not something I'm super concerned about, just trying to make potential future logic checks based on version possible. Since messages are ephemeral though... maybe not worth the effort?

Reply:
I don't really understand.  Even if you want to keep the symbol name constant over upgrades, can't you do this?

```
const (
   foov1 = "foo.v1"
   FOO  = foov1
)
```

I'm always hesitant to use exported `var`s, and I don't understand what it buys us here.

Reply:
Fixed with const's